libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -67,6 +67,14 @@ const AP_Param::GroupInfo AP_LandingGear::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("RETRACT_ALT", 8, AP_LandingGear, _retract_alt, 0),
 
+    // @Param: OPTIONS
+    // @DisplayName: Landing gear auto retract/deploy options
+    // @Description: Options to retract or deploy landing gear in Auto or Guided mode
+    // @Values: 1:Retract after Takeoff, 2:Deploy during Land, 3:Retract after Takeoff AND deploy during Land
```

Comment:
Make this frame-specific if we're not going to do Plane.

Reply:
I've raised an issue to add support for Plane as well. https://github.com/ArduPilot/ardupilot/issues/13665.  I think we will do this in the near future so I'll skip temporariliy making the param invisible for non-Copter frames